pandora-tests/test/utils/server.ts

```diff
@@ -86,14 +37,35 @@ const DIRECTORY_ENV_DEFAULTS: DirectoryEnvSetup = {
 export interface TestStartDirectoryOptions {
 	/** Whether to keep the server active across tests */
 	keepActive: boolean;
-	configOverrides: Partial<DirectoryEnvSetup>;
+	configOverrides: DirectoryEnvSetup;
```

Comment:
This should still be only partial, as it overrides the already complete set in `DIRECTORY_ENV_DEFAULTS`.